test/test_build.py

```diff
@@ -81,17 +81,23 @@ def image_type_fixture(shared_tmpdir, build_container, request, force_aws_upload
     container_ref, images, target_arch, local = parse_request_params(request)
 
     if local:
-        cont_tag = "localhost/cont-base-" + "".join(random.choices(string.digits, k=12))
+        container_ref = "localhost/cont-base-" + "".join(random.choices(string.digits, k=12))
 
         # we are not cross-building local images (for now)
-        request.param = ",".join([cont_tag, images, "", "true"])
+        request.param = ",".join([container_ref, images, "", "true"])
 
-        # copy the container into containers-storage
-        subprocess.check_call([
-            "skopeo", "copy",
-            f"docker://{container_ref}",
-            f"containers-storage:[overlay@/var/lib/containers/storage+/run/containers/storage]{cont_tag}"
-        ])
+    # copy the container into containers-storage
+    # for all builds
+    arch = "amd64"
+    if target_arch == "aarch64" or target_arch == "arm64":
```

Comment:
Hm, we probably want to raise an exception here if we come accross a case when we hit an unknown arch, just to be on the safe side.

Reply:
If I understand correctly, @ondrejbudai was keen to wait until we can have tagged releases before merging this. So I could potentially do the changes now

Reply:
I went a bit defensive and raised an error here but passed down the target arch as is to podman